pkg/server/routes.go

```diff
@@ -28,6 +28,7 @@ func ApiRouter(router *mux.Router, notificationChannel chan common.Notification)
 	router.HandleFunc("/hosts", HostsHandler)
 	router.HandleFunc("/long", LongRunningTaskHandler(notificationChannel))
 	router.HandleFunc("/bootstrap-vm", BootstrapVMHandler(notificationChannel))
+	router.HandleFunc("/create-ignition-configs", CreateIgnitionConfigsHandler)
```

Comment:
Having "create" in the path doesn't seem all that ReSTful, should we perhaps just GET api/ignition/configs, or optionally POST to ignition-configs with an install-config.yaml as the payload?

Reply:
Yes, good point.

Reply:
Done